appium/webdriver/webelement.py

```diff
@@ -204,3 +205,16 @@ def set_value(self, value):
         }
         self._execute(Command.SET_IMMEDIATE_VALUE, data)
         return self
+
+    def send_keys_direct(self, *value):
```

Comment:
I'd rather overload the original selenium's implementation of send_keys. We don't support sending files in neither of Appium drivers.

Reply:
hm, for mobile browser, too? I considered browser cases. (I know we don't support it for native apps)

I found Ruby binding handles the behaviour with a flag, so I wondered if we also do a similar thing for send_keys if mobile safari/chrome can work as same as desktop browser.

Reply:
Safari mobile driver for sure does not support local files upload. Let me check chromedriver sources

Reply:
Chromedriver handles the path inside sendKeys handler: https://github.com/bayandin/chromedriver/blob/47f9916496782bf8c1a031a3c1f1f0c7cf654da1/element_commands.cc#L452

This means there is also no need to modify/patch the call on the client side

Reply:
We've never declared file upload support anyway: https://github.com/appium/appium/issues/13156

However, in case of mobile Chromedriver (at least according to what is present in its code) it might be possible to provide a remote file path into `sendKeys` endpoint and in case the target input is of type `file` and the remote path exists (the file must be uploaded to the device and located somewhere, where it would accessible for the browser or webview owner process) the feature should work

Reply:
Got it 👍 
Thanks for the investigation